wall/wall.c

```diff
@@ -70,14 +69,15 @@ void main(int argc, char *argv[]) {
 	strncpy(sterm, ttyname(fileno(stderr)), sizeof(sterm));
 	for (i = 1; sterm[i] != '/'; i++);
```

Comment:
This may overrun if there are no slashes in the string. You may like to use `for (i = 1; i < sizeof(sterm) && sterm[i] != '/'; i++)`.

Reply:
O.k., valid.

Reply:
Done.